hieradata/class/production/jenkins.yaml

```diff
@@ -11,6 +11,7 @@ govuk_jenkins::job_builder::jobs:
   - govuk_jenkins::jobs::check_pingdom_ip_ranges
   - govuk_jenkins::jobs::content_audit_tool
   - govuk_jenkins::jobs::content_performance_manager
+  - govuk_jenkins::jobs::copy_data_from_production_to_aws
```

Comment:
Didn't we decide to copy data from Staging to Staging AWS, to avoid locking database tables in production?

Reply:
Or is this the Production AWS data sync?

Reply:
Yes, this is from production carrenza to production aws.